gson/src/test/java/com/google/gson/internal/ConstructorConstructorTest.java

```diff
@@ -42,7 +42,7 @@ private interface Interface {}
   public void testGet_AbstractClassNoArgConstructor() {
     ObjectConstructor<AbstractClass> constructor =
         constructorConstructor.get(TypeToken.get(AbstractClass.class));
-    var e = assertThrows(RuntimeException.class, () -> constructor.construct());
+    var e = assertThrows(RuntimeException.class, constructor::construct);
```

Comment:
Unless Error Prone is making you do this, I think this code should probably be left the way it is. It's asserting that the operation `constructor.construct()` throws an exception, and I think `() -> constructor.construct()` is a clearer way to express that.